operate/importer-8_5/src/main/java/io/camunda/operate/zeebeimport/processors/FlowNodeInstanceZeebeRecordProcessor.java

```diff
@@ -53,27 +57,59 @@ public class FlowNodeInstanceZeebeRecordProcessor {
   private static final Logger LOGGER =
       LoggerFactory.getLogger(FlowNodeInstanceZeebeRecordProcessor.class);
   private static final Set<String> AI_START_STATES = Set.of(ELEMENT_ACTIVATING.name());
-
+  private final ConcurrentInitializer<FNITransformer> fniTransformerLazy;
```

Comment:
Implementation looks good to me. Must also be applied to the 8.4 `FlowNodeInstanceZeebeRecordProcessor` class.